web/template/change-password.gotmpl

```diff
@@ -0,0 +1,49 @@
+{{ template "page" . }}
+
+{{ define "title" }}Change password{{ end }}
+
+{{ define "main" }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Change password</h1>
+
+      {{ if .Error }}
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+          <h2 class="govuk-error-summary__title" id="error-summary-title">There was a problem</h2>
+
+          <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+              <li>{{ .Error }}</li>
```

Comment:
Per [Design System](https://design-system.service.gov.uk/components/error-summary/#linking-from-the-error-summary-to-each-answer) "If you do not know which field contains an error, link to the first field."
```suggestion
              <li>
                <a href="#f-currentpassword">{{ .Error }}</a>
              </li>
```

Reply:
I have obviously never really read this page, this is weird advise but OK, also spotted

> As well as showing an error summary, follow the validation pattern - for example, by adding ‘Error: ’ to the beginning of the page <title> so screen readers read it out as soon as possible.

Reply:
Good catch, I've read that a bunch of times and never done it.